docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-to-kubernetes-version-1.25.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Migrate to Kubernetes version 1.25"
+menuTitle: "Migrate to Kubernetes 1.25"
+description: "Learn how to migrate the Helm chart to Kubernetes version 1.25"
+weight: 10
+---
+
+# Migrate to Kubernetes version 1.25
+
+This procedure describes how to prepare a Grafana Mimir Helm chart deployment for an upgrade to Kubernetes 1.25.
+
+## Background
+
+Kubernetes version 1.25 removes the support for the deprecated [PodSecurityPolicy](https://kubernetes.io/docs/concepts/security/pod-security-policy/) object. You can learn more about this topic by visiting [PodSecurityPolicy Deprecation: Past, Present, and Future](https://kubernetes.io/blog/2021/04/06/podsecuritypolicy-deprecation-past-present-and-future/).
+
+Due to how Helm [works](https://helm.sh/docs/topics/kubernetes_apis/), PodSecurityPolicy objects must already be removed from the deployment on Kubernetes version 1.24. If you are using PodSecurityPolicy on Kubernetes 1.24, this is a breakind change. Grafana Mimir helm chart versions before 5.0 used PodSecurityPolicy by default.
```

Comment:
```suggestion
Due to how Helm [works](https://helm.sh/docs/topics/kubernetes_apis/), PodSecurityPolicy objects must already be removed from the release prior to upgrading to Kubernetes version 1.25. If you are using PodSecurityPolicy on Kubernetes 1.24, this is a breaking change. Grafana Mimir helm chart versions before 5.0 used PodSecurityPolicy by default in Kubernetes version 1.24.
```

* Isn't "release" a more accurate term? I think that's what each different installation is called


* Also technically it shouldn't be removed in 1.24, but on whatever release was prior to the upgrade to 1.25.
* Also last point about making it clear what the behaviour was - it was by default in <=1.24